packages/next/src/plugins/plugin.ts

```diff
@@ -152,6 +155,18 @@ function getStartTargetConfig(options: NextPluginOptions, projectRoot: string) {
   return targetConfig;
 }
 
+function getStaticServeTargetConfig(options: NextPluginOptions) {
+  const targetConfig: TargetConfiguration = {
+    executor: '@nx/web:file-server',
+    options: {
+      buildTarget: options.buildTargetName,
```

Comment:
i am looking at the generated project.json with serve target, should the buildTarget be projectName:options.buildTargetName?

Reply:
I am not sure I understand but it follows the same principle as `serveTarget` inside the `plugin.ts`